### PR TITLE
NEVISACCESSAPP-5565: Update example app to use SDK 3.6.0

### PR DESCRIPTION
**Pull request by tamas-toth**
- Updated iOS SDK to version 3.6.0
- Fixed an issue when using device passcode user verifier
- Modified `PinPolicy` comments

### CODE REVIEW

File: NevisExampleApp/Domain/Use Case/DeleteAuthenticatorsUseCaseImpl.swift [hunk 1]
```diff
@@ -42,7 +42,7 @@ extension DeleteAuthenticatorsUseCaseImpl: DeleteAuthenticatorsUseCase {
 		}
 
 		var responses: [Observable<OperationResponse>] = []
-		accounts.forEach { account in
+		for account in accounts {
```
Comment (viktor-gulyas) [inline NevisExampleApp/Domain/Use Case/DeleteAuthenticatorsUseCaseImpl.swift:45]: I think the SwiftFormat rule is missing here not to rewrite the `forEach`

File: NevisExampleApp/Domain/Use Case/DeregistrationUseCaseImpl.swift [hunk 2]
```diff
@@ -41,7 +41,7 @@ extension DeregistrationUseCaseImpl: DeregistrationUseCase {
 		}
 
 		var responses: [Observable<OperationResponse>] = []
-		authenticators.forEach { authenticator in
+		for authenticator in authenticators {
```
Comment (viktor-gulyas) [inline NevisExampleApp/Domain/Use Case/DeregistrationUseCaseImpl.swift:44]: I think the SwiftFormat rule is missing here not to rewrite the forEach

File: NevisExampleApp/Domain/Validators/AccountValidatorImpl.swift [hunk 3]
```diff
@@ -19,7 +19,7 @@ extension AccountValidatorImpl: AccountValidator {
 		}
 
 		var accounts = [any Account]()
-		supportedAuthenticators.forEach { authenticator in
+		for authenticator in supportedAuthenticators {
```
Comment (viktor-gulyas) [inline NevisExampleApp/Domain/Validators/AccountValidatorImpl.swift:22]: I think the SwiftFormat rule is missing here not to rewrite the forEach

**Review by viktor-gulyas — APPROVED** [review]

File: NevisExampleApp/Domain/Use Case/DeleteAuthenticatorsUseCaseImpl.swift (hunk 1 above)
Comment (tamas-toth) [reply NevisExampleApp/Domain/Use Case/DeleteAuthenticatorsUseCaseImpl.swift:45]: Fixed.

File: NevisExampleApp/Domain/Validators/AccountValidatorImpl.swift (hunk 3 above)
Comment (tamas-toth) [reply NevisExampleApp/Domain/Validators/AccountValidatorImpl.swift:22]: Fixed.

File: NevisExampleApp/Domain/Use Case/DeregistrationUseCaseImpl.swift (hunk 2 above)
Comment (tamas-toth) [reply NevisExampleApp/Domain/Use Case/DeregistrationUseCaseImpl.swift:44]: Fixed.

**Pull request merged**